examples/networking/subnet/main-customdns.tf

```diff
@@ -0,0 +1,6 @@
+resource "vkcs_networking_subnet" "custom_subnet_1" {
+  name       = "custom_subnet_1"
```

Comment:
follow name rules, pls

Reply:
renamed objects so that name contains tf-example and meaning of example